specs/sb/api.md

```diff
@@ -296,6 +296,64 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
   <td>boolean</td>
   <td>This field allows the plan to be limited by the non_basic_services_allowed field in a Cloud Foundry Quota, see <a href="http://docs.cloudfoundry.org/running/managing-cf/quota-plans.md">Quota Plans</a>. Default: true</td>
 </tr>
+<tr>
+  <td>&nbsp;&nbsp;&nbsp;schemas</td>
+  <td>object</td>
+  <td>The schema definitions for the input and output properties for service instances and bindings of the plan.</td>
+</tr>
+</tbody>
+</table>
+
+<h5> Schemas Object <a name="PObject"></a> </h5>
```

Comment:
Doug, I was going for consistency with how the rest of the doc is. For example, a Plan Object is a sub-section of the Service Object, and so the Schemas Object acts in the same manner.